### ISSUE TITLE
`$$tag:contains()` breaks sites

### ISSUE THREAD

**Issue by Alex-302** [AdguardTeam/AdguardBrowserExtension#2826]
### Please answer the following questions for yourself before submitting an issue

- [X] Filters were updated before reproducing an issue
- [X] I checked the [knowledge base](https://adguard.com/kb/) and found no answer
- [X] I checked to make sure that this issue has not already been filed

### AdGuard Extension version

4.3.46

### Browser version

FireFox 126.0.1 (64-bit

### OS version

Windows 10 x64

### Ad Blocking

AdGuard Base filter

### Issue Details

Steps to reproduce:
1. Add `carscoops.com$$script:contains(error-report.com)` to user rules
2. Open carscoops.com

Then disable this rule and check `carscoops.com$$script[tag-content="error-report.com"]`

Note: `$$` rules can work only in Firefox.

Reported by @JobcenterTycoon at https://github.com/AdguardTeam/AdguardFilters/pull/180555#issuecomment-2142922776

### Expected Behavior

Site works.

### Actual Behavior

Site is hanging.

**Comment by scripthunter7** [comment]
Its already fixed in https://github.com/AdguardTeam/AdguardBrowserExtension/issues/2646

And we add `:contains` a bit later: https://github.com/AdguardTeam/AdguardBrowserExtension/issues/3150

**Comment by Alex-302** [comment]
@scripthunter7 @ameshkov How about revising the terms? This prevents blocking a popular script that bypasses ad blocking (A-S).

**Comment by maximtop** [comment]
> [@scripthunter7](https://github.com/scripthunter7) [@ameshkov](https://github.com/ameshkov) How about revising the terms? This prevents blocking a popular script that bypasses ad blocking (A-S).

what prevents?

this?
```
And we plan to add :contains a bit later: https://github.com/AdguardTeam/AdguardBrowserExtension/issues/3150
```

**Comment by Alex-302** [comment]
@maximtop Yes. Now we need to use `$replace` rules in FF instead of `$$script:contains()`

**Comment by scripthunter7** [comment]
i've closed this issue, because its already fixed in https://github.com/AdguardTeam/tsurlfilter/commit/cb372ed830bbbb93edaffa7f8a5eb5570bca500c

(i mean, we fixed the breakage, and will add support for `:contains` a bit later, but we have a separate task for it)
